src/lightning/pytorch/utilities/types.py

```diff
@@ -84,6 +85,20 @@ class LRSchedulerConfig:
     strict: bool = True
 
 
+@dataclass
```

Comment:
Could also use TypedDict

Reply:
In this format, the user returns a dictionary, not a dataclass. Same for the lr-scheduler config.

Reply:
So you're saying I should use TypedDict for both LRSchedulerConfig and OptimizerLRSchedulerConfig?